platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/Mapbox.java

```diff
@@ -87,6 +90,11 @@ public static void setAccessToken(String accessToken) {
     FileSource.getInstance(getApplicationContext()).setAccessToken(accessToken);
   }
 
+  public static String getSkuToken() {
+    validateMapbox();
+    return INSTANCE.accounts.getSkuToken();
```

Comment:
If a developer doesn't provide an access token, eg. when using their own tile endpoint, this will throw a null pointer exception when the SDK tries to make a request.

Reply:
This method is only called for Mapbox API URLs so it's safe to remove the validation.